docs/reference/schemas/knowledge_panels/elements/text_element.yaml

```diff
@@ -7,8 +7,14 @@ description: |-
 
   For some specific texts that correspond to a product field (e.g. a product name, the ingredients list of a product),the edit_field_* fields are used to indicate how to edit the field value.
 properties:
-  type:
+  text_type:
```

Comment:
There is discrepancy with the implementation in Dart, the knowledge panels in the web site just use "type", not "text_type".

https://github.com/openfoodfacts/openfoodfacts-server/blob/main/templates/web/panels/panel.tt.html#L154

and the values implemented server side are those:

```
                [% IF text_element.type == "h1" %]
                    <h4 class="panel_title">
                [% ELSIF text_element.type == "quote" %]
                    <blockquote class="panel_text panel_text_quote">
                [% ELSIF text_element.type == "note" %]
                    <div class="panel_text panel_text_note">→
                [% ELSIF text_element.type == "warning" %]
                    <div class="panel_text panel_text_warning">⚠️
                [% ELSIF text_element.type == "callout_info" %]
                    <div class="panel_text alert-box info">
                [% ELSIF text_element.type == "callout_warning" %]
                        <div class="panel_text alert-box warning">⚠️
                [% ELSE %]
                    <div class="panel_text">
                [% END %]
```